### PR TITLE
Add support for Lambda functions

### PR DESCRIPTION
**Pull request by reedloden**

### CODE REVIEW

File: internal/awsdata/lambda.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+		d.results <- result{
+			Row: inventory.Row{
+				UniqueAssetIdentifier:          aws.StringValue(f.FunctionName),
+				Virtual:                        true,
+				BaselineConfigurationName:      aws.StringValue(f.Version),
+				OSNameAndVersion:               "Amazon Linux",
+				Location:                       region,
+				AssetType:                      AssetTypeLambdaFunction,
+				HardwareMakeModel:              fmt.Sprintf("%ds, %dMB", aws.Int64Value(f.Timeout), aws.Int64Value(f.MemorySize)),
```
Comment (reedloden) [inline internal/awsdata/lambda.go]: Taking a bit of an artistic license to this column... trying to provide some useful information about the specific resource. Welcome thoughts...

**Comment by reedloden** [comment]
@itmecho This one has been rebased and ready for review. Pagination testing isn't here, but I will fix it across the board once we standardize on a testing method.

**Comment by itmecho** [comment]
@reedloden sorry for the rebase again, I merged in your fmt/lint fixes and added steps to the CI to enforce these standards so we can make sure all the PRs meet that quality level. I also merged 2 dependabot PRs to bring the libraries up to date

**Comment by reedloden** [comment]
@itmecho ok, rebased again.

**Review by itmecho — APPROVED** [review]

**Comment by itmecho** [comment]
Another great addition :+1:

**Pull request merged**